docs/UserGuide.md

```diff
@@ -170,6 +173,955 @@ may have typed a command incorrectly as **RC4HDB** will warn you in the event th
+**RC4HDB**.
+
+## Keyboard shortcuts
+
+To encourage keyboard usage, here are several keyboard shortcuts that you may find useful for your use:
+
+| Key        | Functionality                                  |
+|------------|------------------------------------------------|
+| `Esc`      | Closes the Help Window if it's open            |
+| `F1`       | Opens the Help Window if it's not open         |
+| `F2`       | Highlights the first row of the displayed list |
+| `F3`       | Quick-access to the command input box          |
+| `Ctrl+Tab` | Switch between Resident and Booking tabs       |
+
+This feature is especially useful if you want to refer to our Help Window which contains a summary of our most basic
+commands. If you want a full summary of all our commands, refer to [here](#command-summary).
+
+[↑ Back to Top](#welcome-to-rc4hdb-user-guide)
+
+---
+
+### Accessing command history
+
+Use the `↑ Up` and `↓ Down` arrow keys to navigate through the command history. This
+is useful if your next command is similar to a previous command, or if you wish to revisit
+your previous commands.
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:**
+Only valid commands will be saved in the history!
+</div>
+
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** Any valid command typed but not entered
+will not be saved!. </div>
+
+[↑ Back to Top](#welcome-to-rc4hdb-user-guide)
+
+---
+
+## **Exiting the Program**
+
+You do not have to worry about exiting
+**RC4HDB** without saving as any changes you've made is automatically saved after every command. For more information,
+the [file management section](#saving-the-data).
+
+### Exiting the program : `exit`
+
+Exits the program.
+
+Format: `exit`
+
+[↑ Back to Top](#welcome-to-rc4hdb-user-guide)
+
+---
+
+## **Upcoming Features**
+
+This section lists some of our upcoming features that we are excited to bring to you!
+
+### Exporting to CSV file : `file export`
+
+As important as it is to be able to import files, we also understand that you might have the need to export them.
+**RC4HDB** will soon provide users with the ability to export data to `.csv` files.
+The file will be safe to remove from the `[JAR file location]/data` folder.
+
+Format: `file export FILE_NAME`
+
+:information_source: The csv file will be exported in this [format](#csv-file-format).<br>
+
+Examples:
+* `file export residents` will export the data from `residents.json` file into a `.csv` file named `residents.csv`.
+
+### *More coming soon!*
+
+[↑ Back to Top](#welcome-to-rc4hdb-user-guide)
+
+---
+
+## **FAQ**
+
+**Q: Can I search using fields other than the name?**
+
+**A**: You can use the `filter` command to search for people using the other fields.
+
+**Q:** Why is there a need to use `hideonly` if we can already use `showonly`?
+
+**A**: You can use `hideonly` when you want to display more fields than you want to hide. For example,
```

Comment:
All of the Q&A on `showonly` `hideonly` could be a bit too detailed and technical for the user. We can consider removing.

Reply:
Hi Neale, the section on FAQ has already been updated based on the comments from CS2101.